prusti/tests/extern_spec/pass/swap.rs

```diff
@@ -0,0 +1,28 @@
+#![feature(type_ascription)]
```

Comment:
Is this used (in the generated code)?

Reply:
It is needed for this particular test case